src/configManager.ts

```diff
@@ -0,0 +1,93 @@
+import { ProductDetails } from './product';
+import { SdkOptions } from './sdkOptions';
+
+type storedConfig = Pick<
+  SdkOptions,
+  'baseUrl' | 'useCookie' | 'refreshRetryPeriod' | 'cookiePath' | 'cookieDomain'
+>;
+
+const getConfigFromSdkOptions = (options: SdkOptions): storedConfig => {
+  const config: storedConfig = {
+    refreshRetryPeriod: options.refreshRetryPeriod,
+    baseUrl: options.baseUrl,
+    useCookie: options.useCookie,
+    cookiePath: options.cookiePath,
+    cookieDomain: options.cookieDomain,
+  };
+  return config;
+};
```

Comment:
Do we know we are guaranteed to have these params, since in SdkOptions, since none of these are required in the type definition

Reply:
They can be null.  it will just store the property as null in the cookie/localstorage.